gui/src/components/modalServer.vue

```diff
@@ -808,11 +822,13 @@ export default {
           alpn: u.params.alpn || "",
           sni: u.params.sni || "",
           tls: u.params.security || "none",
+          quicSecurity: u.params.quicSecurity || "none",
```

Comment:
Does none as the default value work?
I noticed that key should be encoded, could you please take a testing for 'key'? Thank you.